test/test_web.py

```diff
@@ -4131,6 +4130,7 @@ def test_45_password_reset_link_json(self, signer, queue, mock_url):
         db.session.add_all([jane, google, facebook])
         db.session.commit()
 
+        # queue contains msgs for each of the types, accesses them by index
```

Comment:
looks like no changes to this file except comments and blank lines removed. can be reverted to current state